server/api/handlers/query/querymodel.py

```
@@ -4,57 +4,84 @@
+    else:
+        logging.info(f"Successfully updated chat history for user: {username}")
+    
+    return chat_entry
 
+def get_chat_history(username):
+    """Retrieve chat history for a specific user from MongoDB. If empty, initialize with a welcome message."""
+    user = users_chat.find_one({'username': username}, {'chat_history': 1})
+    
+    if user and 'chat_history' in user:
+        # Check if chat history is empty
+        if not user['chat_history']:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)